src/layout/PDFPreviewButton/PDFPreviewButtonComponent.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import type { PropsFromGenericComponent } from '..';
+
+import { PDFGeneratorPreview } from 'src/components/PDFGeneratorPreview/PDFGeneratorPreview';
+import { useStrictInstanceId } from 'src/features/instance/InstanceContext';
+import { NodesInternal } from 'src/utils/layout/NodesContext';
+import { useNodeItem } from 'src/utils/layout/useNodeItem';
+import type { NodeValidationProps } from 'src/layout/layout';
+
+export type IActionButton = PropsFromGenericComponent<'PDFPreviewButton'>;
+
+export function PDFPreviewButtonRenderLayoutValidator({ node }: NodeValidationProps<'PDFPreviewButton'>) {
+  const instanceId = useStrictInstanceId();
+
+  const addError = NodesInternal.useAddError();
+
+  if (!instanceId) {
+    const error = `Cannot use PDF preview button in a stateless app`;
+    addError(error, node);
+    window.logErrorOnce(`Validation error for '${node.id}': ${error}`);
+  }
+  return null;
+}
```

Comment:
This component requires a minimum backend version to work in production, so I think we should also verify that the backend version is at least the required version and error the component otherwise. Or we could risk people using it with an older backend version, and they would not run into any problems before releasing to prod, so they would not notice until its too late